report_py3o/migrations/13.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr, """SELECT id, py3o_template_data FROM py3o_template"""
+    )
+    templates = env.cr.fetchall()
+    for template_id, data in templates:
+        template = env["py3o.template"].browse(template_id)
+        template.write({"py3o_template_data": bytes(data)})
```

Comment:
Should we drop the `py3o_template_data` column from the `py3o_template` table? Nevertheless, it seems that this kind of task into the migration is covered by the Odoo's migration scripts.

Reply:
I prefer to not drop columns as they could be used later on other scripts.
I think that with clean database we could find this lost fields